java-frontend/src/main/java/org/sonar/java/se/symbolicvalues/BinarySymbolicValue.java

```diff
@@ -22,25 +22,55 @@
 import com.google.common.base.Preconditions;
 import com.google.common.collect.ImmutableList;
 
+import org.sonar.java.se.ProgramState;
+import org.sonar.plugins.java.api.semantic.Symbol;
+
+import javax.annotation.Nullable;
+
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.stream.Stream'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)